src/common/EIP712/RelayRequest.ts

```diff
@@ -8,20 +8,7 @@ export default interface RelayRequest {
 
 export function cloneRelayRequest (relayRequest: RelayRequest): RelayRequest {
   return {
-    request: {
-      to: relayRequest.request.to,
-      data: relayRequest.request.data,
-      from: relayRequest.request.from,
-      nonce: relayRequest.request.nonce,
-      gas: relayRequest.request.gas
-    },
-    relayData: {
-      gasPrice: relayRequest.relayData.gasPrice,
-      pctRelayFee: relayRequest.relayData.pctRelayFee,
-      baseRelayFee: relayRequest.relayData.baseRelayFee,
-      paymaster: relayRequest.relayData.paymaster,
-      forwarder: relayRequest.relayData.forwarder,
-      relayWorker: relayRequest.relayData.relayWorker
-    }
+    request: { ... relayRequest.request },
+    relayData: { ... relayRequest.relayData }
```

Comment:
I honestly prefer the verbose, type-aware way...

Reply:
it caused so much hassle to fix on each change. I honestly want a "deep-clone" here. I settled on using one-level clone.

Reply:
This PR signifies an API freeze, so... Kinda...